docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,565 @@
-    PropertyMap imageVisualTransform = new PropertyMap();
-    imageVisualTransform.Add((int)VisualTransformPropertyType.Offset, new PropertyValue(new Vector2(10.0f, 0.0f)))
-                        .Add((int)VisualTransformPropertyType.OffsetPolicy, new PropertyValue(new Vector2((int)VisualTransformPolicyType.Absolute, (int)VisualTransformPolicyType.Absolute)))
-                        .Add((int)VisualTransformPropertyType.SizePolicy, new PropertyValue(new Vector2((int)VisualTransformPolicyType.Absolute, (int)VisualTransformPolicyType.Absolute)))
-                        .Add((int)VisualTransformPropertyType.Size, new PropertyValue(new Vector2(40.0f, 40.0f)))
-                        .Add((int)VisualTransformPropertyType.Origin, new PropertyValue((int)Visual.AlignType.CenterBegin))
-                        .Add((int)VisualTransformPropertyType.AnchorPoint, new PropertyValue((int)Visual.AlignType.CenterBegin));
-    _imageVisual.SetTransformAndSize(imageVisualTransform, size);
-    ```
+**Usage:**
+The following example illustrates how to use a `BorderVisual`:
 
-<a name="bordervisual"></a>
-## Border Visual
+   ```csharp
+   BorderVisual _borderVisual = new BorderVisual();
 
-The border visual renders a solid color as an internal border to the control's quad.
+   // Obligatory properties
+   _borderVisual.Color = Color.Red;
+   _borderVisual.BorderSize = 5.0f;
 
-**Figure: Border visual**
+   // Optional properties:
+   // the size of the border will be interpreted as relative
+   _borderVisual.SizePolicy = VisualTransformPolicyType.Relative;
+   // The visual size (width and height) are equal half of the parent size             
+   _borderVisual.RelativeSize = new RelativeVector2(0.5f, 0.5f);
+   // Position interpreted in absolute units
+   _borderVisual.PositionPolicy = VisualTransformPolicyType.Absolute;
+   // The reference point of the Visual is shifted right (50 pixels) and up (100 pixels) from the parent's reference point
+   _borderVisual.Position = new Vector2(50.0f, -100.0f);
+   // The parent's reference point
+   _borderVisual.Origin = Visual.AlignType.BottomBegin;
+   // Point on the parent with respect to which the visual will be placed
+   _borderVisual.AnchorPoint = Visual.AlignType.BottomBegin;
 
-![Border visual](media/border-visual.png)
+   // _visualView is a previously created VisualView
+   _visualView.AddVisual("nameOfBorderVisual", _borderVisual);
+   ```
 
-The following table lists the supported properties. The visual type is `Border` and the visual map is `BorderVisual`.
 
-**Table: Border visual properties**
 
-| Property | Type | Required | Description |
-|----------|------|----------|-------------|
-| `Color` | `Vector4` | Yes |  The color of the border. |
-| `Size` |  `float`  | Yes |   The width of the border in pixels. |
-| `AntiAliasing` | `boolean` | No  |   Whether antialiasing of the border is required. |
+**Figure: BorderVisual after implementing example (_visualView background color set to gray).**
```

Comment:
and like that?

The `BorderVisual` after implementing the above code and setting the `_visualView` background color to gray looks as follows:

I'll fix the descriptions of all visuals according to the scheme that we will establish here. 
I deleted all picture captions and also "Use ...Visual" headings. I left table captions because I think the guide is more transparent with them.

please let me know if you are ok with these changes

Reply:
please check https://github.com/Samsung/tizen-docs/pull/1367#discussion_r539159464

before I change something again, I would like us to come up with one common version first

Reply:
Both styles are acceptable. The point is the concise headings.

I suggested changing headings to sentence because the heading was too long,
and @nawab-samsung suggested making a new sentence from heading to shorten the heading.

So you can use any one of both. 

Also, these styles are mixable.
It means in some places heading can be used, meanwhile sentence description is used at other place.

...

**Figure: AAA**
\!\[AAA image](AAA image link)
...
The result of the preceding code is as follows:
\!\[BBB image](BBB image link)
...